docker/scripts/merge_env.py

```diff
@@ -14,6 +14,13 @@
     "STRUCTURE_TOOL_IMAGE_URL",
     "STRUCTURE_TOOL_IMAGE_TAG",
 ]
+SET_DEFAULT_KEYS = [
+    "DEFAULT_AUTH_USERNAME",
+    "DEFAULT_AUTH_PASSWORD",
+    "SYSTEM_ADMIN_USERNAME",
+    "SYSTEM_ADMIN_PASSWORD",
```

Comment:
@hari-kuriakose we let's comment these out for now

Reply:
@ritwik-g Why? This is required for someone having an existing setup and doing an upgrade.

We will need to have all of these, but use different default creds for default auth and system admin instead, as per the observed issue.

Reply:
@ritwik-g Patched this by setting different default creds for default and admin user.